src/api/app/controllers/webui/staging/projects_controller.rb

```diff
@@ -9,11 +9,10 @@ class ProjectsController < WebuiController
 
       def create
         authorize @staging_workflow
+        staging_project = @staging_workflow.staging_projects.build(name: params[:staging_project_name])
+        authorize(staging_project, :create?)
 
-        staging_project_name = "#{@staging_workflow.project}:Staging:#{params[:staging_project_name]}"
-        staging_project = @staging_workflow.staging_projects.build(name: staging_project_name)
-
-        if staging_project.valid? && staging_project.store
+        if staging_project.save && staging_project.store
```

Comment:
The reason for using `valid?` is because `store` already do a `save!` and `write_to_backend`. So maybe is better use `write_to_backend` instead of `store`. :thinking: